test/libsolidity/syntaxTests/empty_string_var.sol

```diff
@@ -1,11 +1,10 @@
 contract C {
   function f() {
-    var a = "";
+    string memory a = "";
```

Comment:
I'm not sure but perhaps this test case was for testing type deduction.

Reply:
i'm fine with removing this test, but I eliminated already quite a few;